middlewares/authenticator_test.go

```diff
@@ -161,12 +161,12 @@ func TestBasicAuthUserHeader(t *testing.T) {
 		Basic: &types.Basic{
 			Users: []string{"test:$apr1$H6uskkkW$IgXLP6ewTrSuBkTrqE8wj/"},
 		},
-		HeaderField: "X-WebAuth-User",
+		HeaderField: "X-Webauth-User",
```

Comment:
I suppose it was intentionally to lower-case here and that it won't break anything.